docs/system/hardware-metrics.md

```diff
@@ -44,43 +47,8 @@ when creating instruments not explicitly defined in the specification.
 > * SHOULD introduce a control mechanism to allow users to opt-in to the new
 >   conventions once the migration plan is finalized.
 
-## Common hardware attributes
```

Comment:
could you please add link from this document to the common.md#common-attributes and mention that attributes are defined there. Attributes declared here  apply to all metrics in this file and with them removed, this file is not complete.

Reply:
Updated